lib_eio/process.mli

```diff
@@ -140,7 +140,10 @@ val parse_out :
 
 (** {2 Pipes} *)
 
-val pipe : sw:Switch.t -> _ mgr -> [Flow.source_ty | Resource.close_ty] r * [Flow.sink_ty | Resource.close_ty] r
+val pipe : 
+  sw:Switch.t -> 
+  _ mgr -> 
+  [Flow.source_ty | Resource.close_ty] r * [Flow.sink_ty | Resource.close_ty] r
```

Comment:
Now we seem to have lost the change you were trying to fix 😅 ?

The type of this outer (i.e. not the function `pipe` instead the `Pi` interface) should be the relaxed return types.

Reply:
true true. How did I miss that.. I've just rectified that.